lza-infrastructure/landing-zone/server/dev/terragrunt.hcl

```diff
@@ -0,0 +1,61 @@
+include {
+  path = find_in_parent_folders()
+}
+
+locals {
+  common_vars = read_terragrunt_config(find_in_parent_folders("common_vars.hcl"))
+}
+
+generate "dev_tfvars" {
+  path              = "dev.auto.tfvars"
+  if_exists         = "overwrite"
+  disable_signature = true
+  contents          = <<-EOF
+  target_env = "dev"
+  fam_user_pool_name = "dev-fam-user-pool-bcsc"
+  fam_user_pool_domain_name = "dev-fam-user-pool-domain"
```

Comment:
Hi Ian, the user pool name and domain name needs to be unique, so in tools space, we use "lza-tools-fam-user-pool-bcsc" and "lza-tools-fam-user-pool-domain"